tests/v2/validation/nodescaling/scaling_custom_cluster_test.go

```diff
@@ -71,11 +71,11 @@ func (s *CustomClusterNodeScalingTestSuite) TestScalingCustomClusterNodes() {
 		nodeRoles machinepools.NodeRoles
 		client    *rancher.Client
 	}{
-		{"Scaling control plane by 1", nodeRolesControlPlane, s.client},
-		{"Scaling etcd by 1", nodeRolesEtcd, s.client},
-		{"Scaling etcd and control plane by 1", nodeRolesEtcdControlPlane, s.client},
-		{"Scaling worker by 1", nodeRolesWorker, s.client},
-		{"Scaling worker by 2", nodeRolesTwoWorkers, s.client},
+		{"Scaling custom v2prov control plane by 1", nodeRolesControlPlane, s.client},
+		{"Scaling custom v2prov etcd by 1", nodeRolesEtcd, s.client},
+		{"Scaling custom v2prov etcd and control plane by 1", nodeRolesEtcdControlPlane, s.client},
+		{"Scaling custom v2prov worker by 1", nodeRolesWorker, s.client},
+		{"Scaling custom v2prov worker by 2", nodeRolesTwoWorkers, s.client},
```

Comment:
why not get the cluster type after line 82 and update the name before s.Run instead of generically calling it v2prov?

Reply:
Are you going to include the cluster type Markus? Or are you fine with this Caleb?

Reply:
ping'd offline - but yeah I'd like the cluster name included. It won't be here, but you can get the cluster name from the output / clusterObject on line 82

Reply:
Sounds good I'll wait on the update. Thanks Caleb!

Reply:
Resolved